api-participant/src/main/java/bio/terra/pearl/api/participant/controller/PortalController.java

```diff
@@ -46,4 +54,34 @@ public ResponseEntity<Object> listPortalEnvAlerts(String portalShortcode, String
     return ResponseEntity.ok(
         portalDashboardConfigService.findByPortalEnvId(portalEnvironment.getId()));
   }
+
+  @Override
+  @CrossOrigin(
+      origins = {"https://juniperdemodev.b2clogin.com", "https://junipercmidemo.b2clogin.com"},
+      maxAge = 3600,
+      methods = {RequestMethod.GET, RequestMethod.OPTIONS})
+  public ResponseEntity<Object> getBranding(String portalShortcode, String envName) {
+    EnvironmentName environmentName = EnvironmentName.valueOfCaseInsensitive(envName);
+
+    Portal portal =
+        portalService.loadWithParticipantSiteContent(portalShortcode, environmentName, "en").get();
```

Comment:
I'll need to honor the language too, since branding isn't guaranteed to be the same across languages